content/Video_Display.md

```diff
@@ -776,14 +776,14 @@ procedure that waits for accessibility of OAM would be:
 
 The two wait loops ensure that Mode 0 (and Mode 1 if we are at the end
 of a frame) will last for a few clock
-cycles after completion of the procedure. If we need to wait for the VBlank period, it would be
-better to skip the whole procedure, and use a STAT interrupt instead. In any case, using the
-previously mentioned DMA function would be better than writing to OAM directly.
+cycles after completion of the procedure. If we need to wait for the V-Blank period, it would be
+better to skip the whole procedure, and use a STAT interrupt instead. In any case,
+doing a DMA transfer is more efficient than writing to OAM directly.
 
 ::: tip NOTE
 
-When the display is disabled, both VRAM and OAM are accessible at any
-time. The downside is that the screen is blank (white) during this
+When the display is disabled, both VRAM and OAM are accessible.
```

Comment:
```suggestion
While the display is disabled, both VRAM and OAM are accessible.
```
To better highlight that it's a time span, not a single moment.

Reply:
Fixed